ziti/cmd/ziti/cmd/config_templates/controller.yml

```diff
@@ -1,83 +1,37 @@
 v: 3
 
 #trace:
-#  path:                 ctrl.trace
+#  path: "{{ .ZitiCtrlRawname }}.trace"
 
-profile:
-#   cpu:
-#       path: /home/plorenz/tmp/ctrl.cpu.pprof
+#profile:
 #  memory:
 #    path: ctrl.memprof
 
-db: {{.DatabaseFile}}
+db:                     "{{ .ZitiHome }}/db/ctrl.db"
 
 identity:
-  cert: ${ZITI_SOURCE}/ziti/etc/ca/intermediate/certs/ctrl-client.cert.pem
-  server_cert: ${ZITI_SOURCE}/ziti/etc/ca/intermediate/certs/ctrl-server.cert.pem
-  key: ${ZITI_SOURCE}/ziti/etc/ca/intermediate/private/ctrl.key.pem
-  ca: ${ZITI_SOURCE}/ziti/etc/ca/intermediate/certs/ca-chain.cert.pem
+  cert:                 "{{ .ZitiPKI }}/{{ .ZitiCtrlIntermediateName }}/certs/{{ .ZitiCtrlHostname }}-client.cert"
+  server_cert:          "{{ .ZitiPKI }}/{{ .ZitiCtrlIntermediateName }}/certs/{{ .ZitiCtrlHostname }}-server.chain.pem"
+  key:                  "{{ .ZitiPKI }}/{{ .ZitiCtrlIntermediateName }}/keys/{{ .ZitiCtrlHostname }}-server.key"
+  ca:                   "{{ .ZitiPKI }}/cas.pem"
 
-# the endpoint that routers will connect to the controller over.
 ctrl:
-  listener: tls:{{ .ControlPlane.BindAddress }}:{{ .ControlPlane.Port }}
-    #options:
-    # (optional) settings
-    # set the maximum number of connect requests that are buffered and waiting to be acknowledged (1 to 5000, default 1000)
-    #maxQueuedConnects:      50
-    # the maximum number of connects that have  begun hello synchronization (1 to 1000, default 16)
-    #maxOutstandingConnects: 100
-    # the number of milliseconds to wait before a hello synchronization fails and closes the connection (30ms to 60000ms, default: 1000ms)
-  #connectTimeoutMs:       3000
+  listener:             {{if .CtrlListener}}{{ .CtrlListener }}{{else}}tls:0.0.0.0:{{ .ZitiFabCtrlPort }} {{end}}
 
-# the endpoint that management tools connect to the controller over.
 mgmt:
-  listener: tls:{{ .ManagementPlance.BindAddress }}:{{ .ManagementPlane.Port }}
```

Comment:
i'd like to see and keep all the comments and maintain a singular file that has "all the options" in it, even if commented out.

Reply:
Will fix.